static/bidder-info/emtv.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "https://us-east-ep.engagemedia.tv/pserver"
+maintainer:
+  email: "support@engagemedia.tv"
```

Comment:
@EngageMediaHB  Thank you for your contribution to prebid server. We kindly request your assistance in verifying the email address.  An email for verification is sent on `support@engagemedia.tv`. Requesting to respond with "received" message to confirm the email address.

Reply:
>EngageMediaHB Thank you for your contribution to prebid server. We kindly request your assistance in verifying the email address. An email for verification is sent on support@engagemedia.tv. Requesting to respond with "received" message to confirm the email address.

@EngageMediaHB ping on this.

Reply:
@EngageMediaHB re-ping to complete email verification

Reply:
> @EngageMediaHB Thank you for your contribution to prebid server. We kindly request your assistance in verifying the email address. An email for verification is sent on `support@engagemedia.tv`. Requesting to respond with "received" message to confirm the email address.

@EngageMediaHB , please respond back with "received" message to complete  contact email verification

Reply:
@EngageMediaHB , resent verification email

Reply:
received response from `support@engagemedia.tv`